Classes/Command/ReportListCommand.php

```diff
@@ -64,30 +75,26 @@ public function listCommand($format = 'json', $group = 'default')
         if (!isset($GLOBALS['TYPO3_CONF_VARS']['EXTCONF']['json_reports']['output'][$format])) {
             throw new Exception('The output class for format "' . $format . '" has not been configured.', 1517161193);
         }
-        $output = $this->objectManager->get($GLOBALS['TYPO3_CONF_VARS']['EXTCONF']['json_reports']['output'][$format], $result);
-        if (!$output instanceof OutputInterface) {
-            throw new Exception('The output class "' . get_class($output) . '" does not implement OutputInterface.', 1517161194);
+        $outputClass = $GLOBALS['TYPO3_CONF_VARS']['EXTCONF']['json_reports']['output'][$format];
+        if (!class_exists($outputClass)) {
+            throw new \Exception(sprintf('Invalid output class name %s', $outputClass), 1632821882452);
+        }
+        $reportOutput = GeneralUtility::makeInstance($outputClass, $result);
+        if (!$reportOutput instanceof ReportOutputInterface) {
+            throw new Exception('The output class "' . get_class($reportOutput) . '" does not implement OutputInterface.',
```

Comment:
in der meldung sollte es auch "ReportOutputInterface" heissen.

Reply:
Nur ein Alias, deshalb past es.